### DISCUSSION TITLE
Question about database connection pool in Mirth

### DISCUSSION THREAD

**Discussion by FeiTian01** [nextgenhealthcare/connect#4918] [Q&A] [answered]
How to use database connection pooling in mirth transformer, If I write like this, each request will create a new database connection，my channel Source is HTTPListener，it will receive a lot of requests.
```
dbConn = DatabaseConnectionFactory.createDatabaseConnection('com.mysql.cj.jdbc.Driver', 'jdbc:mysql://localhost:3306/test?useSSL=false&useUnicode=true&characterEncoding=UTF-8&serverTimezone=UTC', 'root', '123456');
resultUpdate = dbConn.executeUpdate('......');
```

**Comment by jonbartels** [comment, marked as answer]
Alex Aitougan shared some snippets at https://github.com/nextgenhealthcare/connect/issues/3819 that may help you

My team and I have used these in a limited capacity for lower-volume workloads. We did have to modify them but I don't recall exactly how. These are a good baseline.

**Reply by FeiTian01** [reply]
thanks a lot ! My friends.
